ovn_cluster.sh

```diff
@@ -446,6 +446,39 @@ function start-db-cluster() {
     $SSL_ARGS start_northd
 }
 
+function check_etcd_mode() {
+    if [[ ! -z "$use_etc" ]]; then
+	    true
+    else
+        false
+    fi
+}
+
+function start_etcd() {
```

Comment:
Q: We are going to need a special version of ovsdb to talk with etcd in one side and northd on the other, right? Is that done in a separate function?

Reply:
yes what we have here is the server side I am adding the client side